DS2S META/ViewModels/PlayerViewModel.cs

```diff
@@ -504,10 +574,15 @@ public float[] StoredPos
         public void ToggleSpeedhack() => ChkSpeedhack = !ChkSpeedhack;
         private void SetGameSpeed() => Hook?.SetSpeed((double)_speedHackFactor);
         public void ToggleNoDeath() => ChkNoDeath = !ChkNoDeath;
+
         public void ToggleOHKO() => ChkOHKO = !ChkOHKO;
         public void ToggleRapierOhko() => ChkRapierOHKO = !ChkRapierOHKO;
         public void ToggleFistOhko() => ChkFistOHKO = !ChkFistOHKO;
 
+        public void ToggleInfiniteSpells() => ChkInfiniteSpells = !ChkInfiniteSpells;
+
+        //public void ToggleDisablePartyWalkTimer() => ChkDisablePartyWalkTimer = !ChkDisablePartyWalkTimer;
```

Comment:
Remants of old code? Something we can remove?

Reply:
Yes indeed it can be removed